packages/server/src/api/controllers/row/external.ts

```diff
@@ -42,6 +43,11 @@ export async function handleRequest<T extends Operation>(
 
 export async function patch(ctx: UserCtx<PatchRowRequest, PatchRowResponse>) {
   const source = await utils.getSource(ctx)
+
+  if (sdk.views.isView(source) && helpers.views.isCalculationView(source)) {
```

Comment:
We can't delete rows through a calculation view as well right? We should likely add that to this PR too.